mqml/tests/test_conductresist.py

```diff
@@ -0,0 +1,76 @@
+"""The test file for conductresist.py"""
+import pytest
+from mqml.parameter.conductresist import ConductResist
+from qcodes.instrument.base import Instrument
+import numpy as np
+import warnings
+
+@pytest.fixture(autouse=True)
+def close_all_instruments():
+    """Makes sure that after startup and teardown, all instruments are closed"""
+    Instrument.close_all()
+    yield
+    Instrument.close_all()
+
+def test_get_initial_values():
+    "This tests the initial values of uncalculated parameters in the class"
+    lockin_param1 = 1
+    lockin_param2 = 2
```

Comment:
This seems to still confuse parameters and the values of the parameters?

Reply:
That's right. I will have an another PR for this as I need to change the source file.